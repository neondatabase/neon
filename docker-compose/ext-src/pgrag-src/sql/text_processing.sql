-- Text processing function tests
SELECT rag.markdown_from_html('<p>Hello <i>world</i></p>');

SELECT rag.chunks_by_character_count('the cat sat on the mat', 10, 5);
