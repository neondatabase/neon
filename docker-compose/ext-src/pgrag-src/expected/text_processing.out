-- Text processing function tests
SELECT rag.markdown_from_html('<p>Hello <i>world</i></p>');
 markdown_from_html 
--------------------
 Hello _world_
(1 row)

SELECT rag.chunks_by_character_count('the cat sat on the mat', 10, 5);
       chunks_by_character_count       
---------------------------------------
 {"the cat","cat sat on","on the mat"}
(1 row)

